modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -169,6 +169,10 @@ public Integer getItemsTotal() {
 		return _itemsTotal;
 	}
 
+	public String getItemType() {
```

Comment:
`itemsType` sounds more correct for this attribute than `itemType` to me

Reply:
Ok, I can change it, no problem!